package.json

```
@@ -47,7 +47,7 @@
     "bower": "^1.8.4",
     "chalk": "^2.4.1",
     "compression": "^1.7.2",
-    "connect-mongo": "^2.0.1",
+    "connect-mongodb-session": "2.0.3",
```

```suggestion
    "connect-mongodb-session": "^2.0.5",
```
Else we're missing out on what look to be useful [releases](https://github.com/mongodb-js/connect-mongodb-session/blob/master/History.md).